apps/mull-ui/src/app/components/event-bullet/event-bullet.scss

```diff
@@ -0,0 +1,33 @@
+.event-bullet-container {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  position: relative;
+  padding: 1rem 0;
+}
+
+.event-photo {
+  max-width: 3rem;
+  max-height: 4rem;
```

Comment:
I'm not sure if having different height and width is a good idea. can it cause trouble for portrait of square photos.

Reply:
changing both to 3rem